acceptance/suites/pre_suite/puppet4_compat/80_configure_agents.rb

```diff
@@ -0,0 +1,13 @@
+require 'puppetserver/acceptance/compat_utils'
```

Comment:
I'd be okay with inlining it if you are.  My point here was I wasn't sure if we needed to do any of the steps in this file for Puppet 4 agents - i.e., could this entire file just be deleted?  I was thinking that with the default values for settings that Puppet 4 agents use that these might be irrelevant - but maybe I'm wrong about that.